test/e2e/util/util.go

```diff
@@ -1218,6 +1218,9 @@ func EnsureAdmissionPolicies(t *testing.T, ctx context.Context, mgmtClient crcli
 	}
 	guestClient := WaitForGuestClient(t, ctx, mgmtClient, hc)
 	t.Run("EnsureValidatingAdmissionPoliciesExists", func(t *testing.T) {
+		if !AtLeast(Version418) {
```

Comment:
why not skip the whole test here
https://github.com/openshift/hypershift/blob/bb6974658cee08dac4f39e3c0d324135725dd54f/test/e2e/util/hypershift_framework.go#L153

Reply:
I thought about that but I think we want to use `t.Skip()` if the assertion isn't supported in the version under test.  That way there is a slot for it in the junit/testgrid explicitly showing the assertion is in the code, but it is being skipped.